src/Client.php

```diff
@@ -2,9 +2,11 @@
 
 namespace LNDHub;
 
-use GuzzleHttp;
-use LNDHub\Contracts\LNDHubClient;
+require_once 'vendor/autoload.php';
```

Comment:
how does this work with composer? does every single repository have a `vendor/autoload.php` file?  (because this is not in the repository?)

Reply:
I did not see this in other packages. 
don't we only need to `require_once` the files WITHIN our package?

Reply:
Since we are using GuzzleHttp, which is a composer package. we need to import the composer vendor to access it. 

The composer `vendor/autload.php` gets generate after running `composer install`

Reply:
but I run composer install not in this library but in our case in the wordpress library.

Reply:
my `wp-lightning/vendor/octan/lndhub` folder does not contain any `vender`

Reply:
Makes sense. Just removed it